src/features/Planner/data/mutations.ts

```diff
@@ -96,3 +96,13 @@ mutation deleteBucket($planId: ID!, $bucketId: ID!) {
   }
 }
 `);
+
+export const COMPLETE_PLAN_ITEM = gql(`
+mutation setStatus($id: ID!, $status: PlanItemStatus!, $doneAt: DateTime) {
+  planner {
+    setStatus(id: $id, status: $status, doneAt: $doneAt) {
+      id
```

Comment:
This should retrieve the newly-updated status, so the cache isn't stale. And it should purge the `getRecipeWithEverything` queries, I think?

Reply:
Added a TODO item to tackle this in the future.